definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -6229,5 +6229,11 @@
     "CaseEventID": "attachScannedDocsWithOcr",
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "R"
+  },  {
+    "LiveFrom": "03/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "allowPetitionerToAmend",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
```

Comment:
Might be worth finding out maybe you'd need to at least grant 'R' for the other roles. If thats been considered already then ignore the comment

Reply:
yep, we usually do that. Add "R" role for solicitor, legal advisor (la) and super-user

Reply:
So, add "R" for: caseworker-divorce-solicitor, caseworker-divorce-courtadmin-la and caseworker-divorce-superuser

Reply:
To clarify, in the next commit, i have added 
```
  {
    "LiveFrom": "03/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseEventID": "allowPetitionerToAmend",
    "UserRole": "caseworker-divorce-solicitor",
    "CRUD": "R"
  },
  {
    "LiveFrom": "03/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseEventID": "allowPetitionerToAmend",
    "UserRole": "caseworker-divorce-courtadmin-la",
    "CRUD": "R"
  },
  {
    "LiveFrom": "03/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseEventID": "allowPetitionerToAmend",
    "UserRole": "caseworker-divorce-superuser",
    "CRUD": "CRUD"
  }
```
For the `caseworker-divorce-superuser` I assume that `CRU` should be the roles, because they are a SUPER user.